CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -160,9 +168,18 @@ local function CreateTopBar()
 		return topbarContainer
 	end
 
+	local function ComputeTransparency(opaque)
+		if not topbarEnabled then
+			return 1
+		elseif opaque then
+			return TOPBAR_OPAQUE_TRANSPARENCY
+		end
+		return TOPBAR_TRANSLUCENT_TRANSPARENCY
+	end
+
 	function this:SetTopbarDisplayMode(opaque)
-		topbarContainer.BackgroundTransparency = opaque and TOPBAR_OPAQUE_TRANSPARENCY or TOPBAR_TRANSLUCENT_TRANSPARENCY
-		topbarShadow.Visible = not opaque
+		topbarContainer.BackgroundTransparency = ComputeTransparency(opaque)
+		topbarShadow.Visible = topbarEnabled and not opaque
 		this:UpdateBackgroundTransparency()
 	end
 
```

Comment:
I missed this before, but since ForceTopbarOpaque is also called by the hamburger menu we probably need to incorporate this into our topbarEnabled code